alpine-chroot-install

```diff
@@ -115,15 +115,14 @@ ewarn() {
 
 # Writes Alpine APK keys embedded in this script into directory $1.
 dump_alpine_keys() {
-	local dest_dir="$1"
-	local content id line
+	dest_dir="$1"
 
 	mkdir -p "$dest_dir"
 	for line in $ALPINE_KEYS; do
 		id=${line%%:*}
 		content=${line#*:}
 
-		printf -- "-----BEGIN PUBLIC KEY-----\n$content\n-----END PUBLIC KEY-----\n" \
+		printf -- "-----BEGIN PUBLIC KEY-----\n%s\n-----END PUBLIC KEY-----\n" "$content" \
 			> "$dest_dir/alpine-devel@lists.alpinelinux.org-$id.rsa.pub"
```

Comment:
This is okay, but also replace double quotes with single quotes in the first argument.